src/provider/github.js

```diff
@@ -1,65 +1,89 @@
-import { createApolloFetch } from 'apollo-fetch';
+import {createApolloFetch} from 'apollo-fetch';
 
 const uri = 'https://api.github.com/graphql';
 
-const apolloFetch = createApolloFetch({ uri });
+const apolloFetch = createApolloFetch({uri});
 
-apolloFetch.use(({ options }, next) => {
+const parseIssues = (project) => (project && project.data && project.data.repository && project.data.repository.issues)
+  ? project.data.repository.issues
+  : { totalCount: 0 };
+
+apolloFetch.use(({
+  options
+}, next) => {
   if (!options.headers) {
     // eslint-disable-next-line no-param-reassign
     options.headers = {
-      authorization: `bearer ${process.env.REACT_APP_GITHUB_KEY}`,
+      authorization: `bearer ${process.env.REACT_APP_GITHUB_KEY}`
     };
   }
   next();
 });
 
-const totalIssuesGetRequest = project =>
-  new Promise((resolve) => {
-    apolloFetch({
-      query: `query IssueCount($owner: String!, $repositoryName: String!) {
+const totalIssuesGetRequest = project => new Promise((resolve) => {
+  apolloFetch({
+    query: `query IssueCount($owner: String!, $repositoryName: String!) {
         repository(owner: $owner, name:$repositoryName) {
           issues() {
             totalCount
           }
         }
       }`,
-      variables: {
-        repositoryName: project.repository,
-        owner: project.owner,
-      },
-    })
-      .then((response) => {
-        resolve({ name: project.repository, issues: response.data.repository.issues });
-      })
-      .catch(err => resolve(err));
+    variables: {
+      repositoryName: project.repository,
+      owner: project.owner
+    }
+  }).then((response) => {
+    if(response.errors) console.error(response.errors);
+    resolve({
+      name: project.name,
+      repository: project.repository,
+      issues: parseIssues(response),
+    });
+  }).catch(err => {
+    console.error(`Failed to get the total issues for ${project.name}. Error: `, err);
+    resolve({
```

Comment:
if you return an valid object from catch then the it continues on the success path: 

```js
 Promise
  .reject(null)
  .catch( error => 1)
  .then(x=> console.log("success"));
```